src/ext/language-mode.lisp

```diff
@@ -153,8 +154,11 @@
       (uncomment-region)
       (comment-region)))
 
-(defun set-region-point (start end)
-  (cond
+(defgeneric set-region-point (start end global-mode))
+
```

Comment:
I think it would be better to define a new method (e.g. set-region-point-using-global-mode) that takes global-mode as an argument, with set-region-point as the default value. What do you think?

Reply:
Indeed, we can change the name of the function to reflect it, and at this point we can use the global-mode symbol so that's a good option too